src/github_integration/model/commit.py

```diff
@@ -0,0 +1,44 @@
+from typing import Optional
+
+from github import Commit as GitCommit
+
+
+class Commit:
```

Comment:
there is nothing really going on in this class. Why it's even needed? the original `Commit` is doing almost the same things. If you want to just package some properties for the sake of simplicity and pass it to some functions, you probably can simplify all this with a nice little `dataclass`

Reply:
1. I am isolating the GitHub library using this kind of object.
2. There is missing logic for the detection of contributing accounts.